pkg/controller/machineset/machineset_controller.go

```diff
@@ -298,9 +301,12 @@ func (r *ReconcileMachineSet) syncReplicas(ms *clusterv1alpha1.MachineSet, machi
 			machine := r.createMachine(ms)
 			if err := r.Client.Create(context.Background(), machine); err != nil {
 				klog.Errorf("Unable to create Machine %q: %v", machine.Name, err)
+				r.recorder.Eventf(ms, corev1.EventTypeWarning, "FailedCreate", "Failed to create machine %q: %v", machine.Name, err)
 				errstrings = append(errstrings, err.Error())
 				continue
 			}
+			klog.Infof("Created machine %q", machine.Name)
```

Comment:
Here you refer to the machine by name, but above by number: Creating machine %d of %d. I don't know how busy the logs are: is it worthwhile to connect these messages together? Maybe the above message includes the name, or this message includes the index?

Reply:
Good call, I don't think we have to worry about how busy the logs are (yet), but I do like the idea of tying the two messages together a bit more. Updated.